src/client/js/Utils/LocalStorageManager.js

```diff
@@ -0,0 +1,76 @@
+/*globals define, $, window, WebGMEGlobal*/
+/**
+ * TODO: This is work in progress and currently not used anywhere.
```

Comment:
You can disregard this file, it's not used. However I'd like to keep it checked in as it has some example of the $.notify and if we end up using the local storage (either for the commit-queue or user-state) - we have a placeholder for it.